src/vehicle.cpp

```diff
@@ -2995,6 +3007,32 @@ void vehicle::center_of_mass(int &x, int &y) const
     y = round(yf);
 }
 
+point vehicle::pivot_point() const
+{
+    // TODO: a part type that let you override this
+    point p;
+    center_of_mass(p.x, p.y, false);
```

Comment:
Sure. Note that it's also called via valid_wheel_config on every turn for a controlled vehicle, and that in theory most controlled vehicles probably need to recalculate it all the time due to fuel burn (though I don't think fuel mass is currently accounted for). It might help for moving-but-uncontrolled vehicles (collisions..)